packages/vue/src/components/atoms/SfBullets/SfBullets.html

```diff
@@ -0,0 +1,23 @@
+<div class="sf-bullets">
+  <span v-for="(_, index) of inactiveLeft" @click="e => $emit('click', index)">
+    <!--@slot custom icon for inactive bullet -->
+    <slot name="inactive" class="sf-bullet">
+      <div class="sf-bullet"></div>
+    </slot>
+  </span>
```

Comment:
imo would be better to have just one default slot here, current bullet is defined from prop so it's easy to check externally if is the active bullet, wdyt?

Reply:
I prefer to leave both as named slots, it's nice and clear and only require typing extra <code>#inactive</code> for the user.